src/interfaces/excercise.ts

```diff
@@ -0,0 +1,25 @@
+// TypeScript Interface Exercise
```

Comment:
please provide a solution for this as well to have it as reference

Reply:
Done, created an `excercise-solution.ts` file to cover the solution.